crates/esplora/tests/async_ext.rs

```diff
@@ -109,6 +109,27 @@ pub async fn test_update_tx_graph_without_keychain() -> anyhow::Result<()> {
         )
         .await?;
 
+    // Check to see if we have the floating txouts available from our two created transactions'
+    // previous outputs in order to calculate transaction fees.
+    graph_update.full_txs().for_each(|tx| {
+        let calculate_fee_result = graph_update.calculate_fee(tx.tx);
+        let get_tx_result = env.bitcoind.client.get_transaction(&tx.txid, None);
```

Comment:
Is there a situation where `bitcoind` cannot get the tx or the fee of the tx here? It'll be easier to read if we just return the error for `get_transaction` and unwrap the fee.

Reply:
I think we should avoid storing `Result`s when we don't have to. It makes it easier to read if error handling goes directly after getting the result.

> Try `TxGraph::calculate_fee` -> We `expect` no error as we have all prev outputs! -> Store fee value in variable.

Reply:
Because `.for_each` takes in a closure which returns nothing, whereas we want to handle our errors, I think we should avoid using it here. The test returns a Result, might as well use it!